lib/helpers/publizon.ts

```diff
@@ -0,0 +1,9 @@
+import { md5 } from "js-md5"
```

Comment:
Why are we moving this away from the pubhub folder? I think it's nice that it was in the local helper.ts file.